src/PhpFile.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Yiisoft\I18n\Resource;
+namespace Yiisoft\I18n;
```

Comment:
That should be `Yiisoft\I18n\Message`, I think.

Reply:
Not with current setup. https://github.com/yiisoft/i18n-message-php/blob/master/composer.json#L29

Reply:
Yes, should be adjusted as well.

Reply:
https://github.com/yiisoft/docs/blob/master/004-namespaces.md

Reply:
You want to rename from `PhpFile` to `Php` as well?

Reply:
No.

Reply:
You got it. Namespace is now Message